cmd/automountServiceAccountToken.go

```diff
@@ -38,7 +38,7 @@ func checkAutomountServiceAccountToken(result *Result) {
 		return
 	}
 
-	if result.Token != nil && *result.Token && result.SA == "" {
+	if result.Token != nil && *result.Token && (result.SA == "" || result.SA == "default") {
```

Comment:
This is a very minor nitpick but could you split this if statement into multiple lines? Ideally put the `(result.SA == "" || result.SA == "default")` on its own line, so it's more readable?

Reply:
Done!